code/game/atoms.dm

```diff
@@ -756,7 +756,7 @@
 	return blood_dna
 
 /mob/living/carbon/alien/get_blood_dna_list()
-	return list("UNKNOWN DNA" = "X*")
+	return list("UNKNOWN DNA" = get_blood_type("X"))
 
 /mob/living/silicon/get_blood_dna_list()
 	return list("MOTOR OIL" = "SAE 5W-30") //just a little flavor text.
```

Comment:
I don't even think I can fix this

Reply:
Might get some flak for this, but honestly— just make a new blood type for IPCs. I've looked over the code, and honestly there's less work to be done that way and while it's kind of funny to use oil as IPC blood it's probably best to just make a synthetic blood type here. I'd like to ask you to remove all the code for that as well, but just adding a new IPC blood type would be just as fine. Don't forget to add the blood packs to the freezers also because. yeah

tl;dr just make a new blood type for ipcs